website/content/en/highlights/2022-03-22-0-21-0-upgrade-guide.md

```diff
@@ -28,6 +29,32 @@ We cover them below to help you upgrade quickly:
 
 ### Breaking changes
 
+#### Syntax changes for non-VRL paths {#path-syntax-changes}
+
+Previously, there were two different ways to describe paths. VRL (remap transforms) use a newer syntax, while everything
+else in Vector (templating / all transforms except remap) still used an older syntax. This was a constant pain point for users,
+and we have taken some steps to migrate towards the VRL syntax. This is a **breaking change** that may require migration.
+
+The old syntax was very lenient in the characters that were allowed in a field name. It also supported single character escapes.
+The new syntax only allows capital/lowercase letters, numbers, and underscore. Any other character will require the field name to be quoted.
+Quotes around a field name replace single character escaping. This brings the old syntax in line with the newer (VRL) syntax, except for support
+of the "coalescing" operator. The new parser is also faster, and contributed to the increased performance you may see in this release.
```

Comment:
Honestly, I wouldn't call out the coalescing operator. It'll likely confuse people more than needed, and if we do get reports of people attempting to use this, it would be a good sign that people have a use for this, and we might reconsider adding it.